ayon_api/_api.py

```diff
@@ -4049,12 +4053,12 @@ def get_entities_links(*args, **kwargs):
     Args:
         project_name (str): Project where links are.
         entity_type (Literal["folder", "task", "product",
-            "version", "representations"]): Entity type.
+        |    "version", "representations"]): Entity type.
```

Comment:
`|` are for sphinx to not complain about the additional indent. Not too sure why it complained here, seems like a glitch to me.

Reply:
It has to be changed in `server_api.py`, content of `_api.py` is created based on that.